dom.bs

```diff
@@ -1026,24 +1050,52 @@ seen from the definition above, an <a>event listener</a> is a more broad concept
 </dl>
 
 <p>The
-<dfn method for=EventTarget><code>addEventListener(<var>type</var>, <var>callback</var>, <var>capture</var>)</code></dfn>
+<dfn method for=EventTarget><code>addEventListener(<var>type</var>, <var>callback</var>, <var>options</var>)</code></dfn>
 method, when invoked, must run these steps:
 
 <ol>
  <li><p>If <var>callback</var> is null, terminate these steps.
 
+ <li>Let <var>capture</var> and <var>passive</var> be false.
+
+ <li>If <var>options</var> is a boolean, set <var>capture</var> to
+ <var>options</var>.
+
+ <li>If <var>options</var> is a dictionary and <code>{{EventListenerOptions/capture}}</code> is
+ present in <var>options</var> with value true, then set <var>capture</var> to true.
+
+ <li>If <var>options</var> is a dictionary and <code>{{EventListenerOptions/passive}}</code> is
+ present in <var>options</var> with value true, then set <var>passive</var> to true.
```

Comment:
I attempted that [here](https://github.com/RByers/dom/commit/e3a2df3c7df30d68247d758c7f746ed76375b92c#diff-a467e681501e456ce8c1ef31425e1b41R1052) and I agree it looks cleaner, thanks!  I couldn't find another example of returning multiple values, is what I've done precise enough?  Any suggestion for a term better than "normalize" (which could perhaps be confused with Node.normalize)?